plugin_tests/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is this file just for developers' editors? The `add_standard_plugin_tests()` will use the [`.eslintrc.json` for Girder test files](https://github.com/girder/girder/blob/master/clients/web/test/.eslintrc.json).

Speaking of which, the name of the file [should now be `.eslintrc.json`](http://eslint.org/docs/user-guide/configuring#configuration-file-formats).